x/derivatives/types/expected_keepers.go

```diff
@@ -10,6 +10,7 @@ import (
 // AccountKeeper defines the expected account keeper used for simulations (noalias)
 type AccountKeeper interface {
 	GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.AccountI
```

Comment:
this method isn't used.

Reply:
I don't want to modify it because I need it when I send it to the ecosystem mod.

Reply:
i don't think it's necessary for that.